index.js

```diff
@@ -111,15 +117,25 @@ class PD {
   }
 
   async _swarmConnection(conn) {
-    conn.once("close", this._connClose);
+    this.store.replicate(conn) // replicate the drives etc
```

Comment:
actually replicate the store lol (this was your bug, you werent connecting any peers in reality, so all lists were empty as thats the state the drive had)